tests/Dfc.CourseDirectory.Core.Tests/ReferenceDataTests/UkrlpSyncHelperTests.cs

```diff
@@ -19,123 +24,111 @@ public UkrlpSyncHelperTests(DatabaseTestBaseFixture fixture)
         }
 
         [Fact]
-        public async Task SyncProviderData_ProviderAlreadyExists_UpdatesProviderInfo()
+        public async Task SyncProviderData_CreatesProvider()
```

Comment:
Nit: most of our tests follow the `MethodName_StateUnderTest_ExpectedBehavior` naming convention so previous naming would be better in this regard

Reply:
Done :+1: